docs/word/dictionary-task-pane-add-ins.md

```diff
@@ -551,22 +620,31 @@ function selectedTextCallback(selectedText) {
 // It also sets the "See More" link.
 function refreshDefinitions(data, textStatus, jqXHR) {
     $(".definition").remove();
+
     // Make a new list item for each returned definition that was returned, set the CSS class, and append it to the definitions div.
     $(data).find("Definition").each(function () {
         $(document.createElement("li")).text($(this).text()).addClass("definition").appendTo($("#definitions"));
     });
+
     // Change the "See More" link to direct to the correct URL.
     $("#SeeMoreLink").attr("href", $(data).find("SeeMoreURL").text());
 }
 
-// This function is called when the add-in gets back the link to the pronunciation
-// to set the "Pronounce" link to the URL of the .WAV file.
-function refreshPronunciation(data) {
-    $("#pronunciationLink").attr("href", data);
-}
-
 // Basic error handler that writes to a div with id='message'.
 function errorHandler(jqXHR, textStatus, errorThrown) {
-    document.getElementById('message').innerText += errorThrown;
+    document.getElementById('message').innerText += ("textStatus:- " + textStatus + "\nerrorThrown:- " + errorThrown + "\njqXHR:- " + JSON.stringify(jqXHR));
 }
 ```
+
+## Try it out
+
+1. Using Visual Studio, test the newly created Word add-in by pressing **F5** or choosing **Debug** > **Start Debugging** to launch Word with the **Show Taskpane** add-in button displayed in the ribbon. The add-in will be hosted locally on IIS.
+
+1. In Word, if the add-in task pane isn't already open, choose the **Home** tab, and then choose the **Show Taskpane** button in the ribbon to open the add-in task pane. (If you're using the volume-licensed perpetual version of Office, instead of the Microsoft 365 version or a retail perpetual version, then custom buttons aren't supported. Instead, the task pane will open immediately.)
```

Comment:
```suggestion
1. In Word, if the add-in task pane isn't already open, choose the **Home** tab, and then choose the **Show Taskpane** button to open the add-in task pane. (If you're using the volume-licensed perpetual version of Office, instead of the Microsoft 365 version or a retail perpetual version, then custom buttons aren't supported. Instead, the task pane will open immediately.)
```
We can eliminate "on the ribbon" in most cases. Once they choose the Home tab, they're already there. so it's redundant.